### DISCUSSION TITLE
Docker Hub - Open-Source program changes

### DISCUSSION THREAD

**Discussion by cksapp** [dani-garcia/vaultwarden#3349] [General]
Probably already getting notice of this, but just curious and wanting to get the conversation going here.

Thoughts on any changes, or how this possibly affects the project? Possibly using GitHub Packages?
[Docker is deleting Open Source organisations - what you need to know](https://blog.alexellis.io/docker-is-deleting-open-source-images/)

**Comment by BlackDex** [comment]
There is no effect for Vaultwarden. We have been accepted by docker for the DSOS project.

But besides that I'm still planning to add GitHub as a second location.

**Comment by BlackDex** [comment]
Also see https://www.docker.com/blog/we-apologize-we-did-a-terrible-job-announcing-the-end-of-docker-free-teams/

**Comment by gabe565** [comment]
I'd love to see GHCR as a secondary location. Other than the recent drama, they also impose some pretty aggressive rate limits that have caused issues for me in the past.
